monocle/tests/unit/test_crawler_gerrit.py

```diff
@@ -46,11 +46,11 @@ def extract_and_compare(self, base_url, name):
 
     def test_extract_and_compare_review1(self):
         """
-        Gerrit crawler extracts https:__gerrit-review.googlesource.com-gerrit-246332
+        Gerrit crawler extracts https__gerrit-review.googlesource.com-gerrit-246332
         """
         self.extract_and_compare(
             "https://gerrit-review.googlesource.com",
-            "https:__gerrit-review.googlesource.com-gerrit-246332",
+            "https__gerrit-review.googlesource.com-gerrit-246332",
```

Comment:
Ok, so please rename it this way gerrit-review.googlesource.com-gerrit-246332 to get the same format (w/o the scheme) than the github.com-*_[extracted|raw].json files.

Thanks for the fix. I wasn't aware that the colons in filenames could cause such troubles on Windows

Reply:
Thank you @morucci for reviewing.. I have addressed your review suggestion.. Hope I understood it correctly. Please me know if it needs any other changes. 👍